src/Api/Soap/Soap.php

```diff
@@ -78,17 +84,16 @@ class Soap extends ApiBase  implements DispatcherAwareInterface
 	 * Method to instantiate the file-based api call.
 	 *
 	 * @param   Container  $container  The DIC object
+	 * @param   Renderer   $renderer   Renderer object.
 	 * @param   Registry   $options    Optional custom options to load
 	 *
 	 * @throws  \Exception
 	 * @since   1.4
 	 */
-	public function __construct(Container $container, Registry $options)
+	public function __construct(Container $container, $renderer, Registry $options)
```

Comment:
You're right.  Must have been from something I was doing earlier.  This code has been largely neglected/forgotten and I'd be very surprised if it actually works at the moment.  I haven't a clue how to test it.